concent_api/core/admin.py

```diff
@@ -24,21 +25,25 @@ def queryset(self, request, queryset):
         elif self.value() == 'passive':
             passive_state_names = [x.name for x in Subtask.PASSIVE_STATES]
             return queryset.filter(state__in=passive_state_names)
+        elif self.value() == 'active_downloads':
+            current_timestamp = get_current_utc_timestamp()
+            return Subtask.objects_with_timing_columns.filter(time_for_download_file__gte=current_timestamp)
         return queryset
 
 
 class SubtaskAdmin(ModelAdminReadOnlyMixin, admin.ModelAdmin):
-
     list_display = [
         'subtask_id',
         'task_id',
         'state',
         'get_provider_public_key',
         'get_requestor_public_key',
         'next_deadline',
+        'computation_deadline',
+        'result_package_size',
```

Comment:
Please show also the three newly added columns. Especially the one that shows the download deadline. This will allow the admin to see how long each task has until the timeout.

Reply:
Done.